Integrations/integration-Intezer.yml

```diff
@@ -0,0 +1,221 @@
+        }
+
+        // create file analysis request
+        var res = sendRequest('POST', HASH_URL, {
+            sha256: hash,
+            code_item_type: codeItemType
+        });
+
+        var analysisId;
+        if (res.statusCode === 201) {
+            analysisId = res.obj.analysis_id;
+        } else {
+            throw 'Failed to create sha256 analysis for ' + hash + ', request status code: ' + res.statusCode + '\n';
+        }
+
+        // perform wait loop
+        res = getAnalysis(analysisId);
+        var tries = 0;
+        while (res.status !== SUCCESS_STATUS && tries < maxRetries) {
+            // this means that status is 'in_progress' or 'queued'
+            tries++;
+            wait(delay); // wait {delay} seconds
+            res = getAnalysis(analysisId);
+        }
+
+        // if polling result did not work, return an error message
+        if (res.status !== SUCCESS_STATUS) {
+            throw 'Failed to pool sha256 analysis for ' + hash + ', Try to change maxRetries or delay arguments\n';
+        } else {
+            // Create new indicator of sha256 according to verdict
+            var verdict = res.result.verdict;
+            var dbotScore = 0;
+            var ec = {};
+            ec[outputPaths.file] = [];
+            if (verdict === 'malicious') {
+                dbotScore = 3;
+                var malFile = {};
+                addMalicious(malFile, outputPaths.file, {
+                    SHA256: hash,
+                    Malicious: {Vendor: 'Intezer', Metadata: res.result}
+                });
+                ec[outputPaths.file].push(malFile[outputPaths.file]);
+            } else if (verdict === 'suspicious') {
+                dbotScore = 2;
+            } else if (verdict === 'trusted' || verdict === 'neutral') {
+                dbotScore = 1;
+            }
+            ec.DBotScore = [{Indicator: hash, Type: 'hash', Vendor: 'Intezer', Score: dbotScore}];
+            var data = res.result;
+            var md = '## Intezer analysis result\n';
+            md += 'SHA256: ' + hash + '\n';
+            md += 'Verdict: **' + data.verdict + '** (' + data.sub_verdict + ')\n';
+            md += '[' + 'Analysis Link' + '](' + data.analysis_url + ')\n';
+            // set return entry with context
+            entries.push({
+              Type: entryTypes.note,
+              Contents: res.body,
+              ContentsFormat: formats.json,
+              HumanReadable: md,
+              EntryContext: ec
+            });
+        }
+
+        return entries;
+    };
+
+    switch (command) {
+        case 'test-module':
+            // send is available
+            return sendRequest('GET', IS_AVAILABLE_URL).obj.is_available;
+        case 'file':
+            return analyzeSHA256(args.file, args.codeItemType, args.maxRetries, args.delay);
+        default:
+    }
+  type: javascript
+  commands:
+  - name: file
```

Comment:
in all argument description - we don't need the type and optional/mandatory at the beginning. optional/mandatory is shown in the interface. type should be derived from the description.